docs/firmware/aws/BuildingUsingDocker.rst

```diff
@@ -52,3 +52,8 @@ Location of the HEX file
 ************************
 
 The built HEX file will be located in :file:`build/zephyr/merged.hex`.
+
+Device credentials
+=====================
+
+Click :ref:`Create device credentials <aws-device-credentials>` to continue with the configuration or click ``Next`` to see other ``firmware`` build options. 
```

Comment:
Hold on, where are we now? What are we clicking and where? If we refer to a doc page, we don't say click. Not that sure we need this section here or any of the docs in this PR.

Reply:
I reviewed the changes, and the point is to bring users back to the Getting Started section, because the navigate outside of it for building the firmware.